packages/lib/actionClass/service.ts

```diff
@@ -55,6 +56,36 @@ export const getActionClasses = reactCache(
     )()
 );
 
+export const getActionClassesForEnvironmentState = reactCache(
```

Comment:
please move this service to the environmentState route segment where its used as we're moving away from the global services approach, so please copy it to `apps/web/app/api/v1/client/[environmentId]/environment/lib/actionClass.ts`